Automattic/jetpack
# DonutMeter Component: Set up a reusable donut chart component

This PR adds a DonutMeter  component to Jetpack reusable components. 

Fixes #25917

#### Changes proposed in this Pull Request:

* Adds `DonutMeter` Component 
* Adds the component to storybook
* Ensures component takes props for adjusting the size, chart thickness, chart segment color, current value, max/total value
* Ensures component has an accessible/screen reader table alternative
* Ensures chart segment begins at the bottom (180 degrees) of the donut and increments in a clockwise direction. 

![image](https://user-images.githubusercontent.com/30754158/187625076-0d2277b7-888b-45d3-9a1e-0e8386dbe351.png)


#### Other information:



#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Run `cd projects/js-packages/storybook && npm run storybook:dev` and confirm that you see DonutMeter component in Storybook.
* Under JS Packages → Components → DonutMeter, choose the story Default and try changing some of the variables under Controls. You should be able to adjust the chart size, donut line width, segment colour, the segment count, and the total count.

You can also run the unit tests for js-packages/components by running `jetpack test`.
